pkg/gofr/service/health_test.go

```diff
@@ -82,7 +82,7 @@ func TestHTTPService_HealthCheckTimeout(t *testing.T) {
 	server := httptest.NewServer(http.HandlerFunc(func(_ http.ResponseWriter, r *http.Request) {
 		assert.Equal(t, "/.well-known/alive", r.URL.Path)
 
-		time.Sleep(2 * time.Second)
+		time.Sleep(1000 * time.Millisecond)
```

Comment:
instead of 1000 millisecond, its more realistic to use 1 sec

Reply:
I was about to say it yes

Reply:
Sure changed it to second